src/shell/output.c

```diff
@@ -84,7 +84,6 @@ struct shell_output {
     struct eventlogger *ev;
```

Comment:
In commit message for 31277bad4117442931d3adcf0e7a6a1981751990 s/refernence/reference/